### ISSUE TITLE
Error when building toolchain

### ISSUE THREAD

**Issue by Chai112** [MandelbrotOS/MandelbrotOS#38]
An error causing the termination of the building of the toolchain.
More specifically, this was using @unsafecast's fork, of build-test branch, PR #37 

To reproduce:
```
$ ./scripts/build-tools.sh
```

![image](https://user-images.githubusercontent.com/41228307/117148656-a2c48600-ade0-11eb-960f-2f44efccfcb8.png)

**Comment by unsafecast** [comment]
Can you please send more of the output?

**Comment by Chai112** [comment]
![image](https://user-images.githubusercontent.com/41228307/117149614-9260db00-ade1-11eb-976b-ceb3f969f89c.png)

**Comment by unsafecast** [comment]
Nevermind i know what's wrong. You should've cloned recursively, like so:

```
git clone --recursive url
```

This is so all the submodules get cloned too. Sorry for the confusion, i should add it to the readme

**Comment by Chai112** [comment]
ah right gotcha ill retry the installation

**Comment by unsafecast** [comment]
You don't need to clone again btw

```
git submodule update --init --recursive
```

is enough

**Comment by Chai112** [comment]
sorry, still the same error even after recloning
![image](https://user-images.githubusercontent.com/41228307/117173887-66e8eb00-adf7-11eb-9906-16e66c82f42b.png)

**Comment by unsafecast** [comment]
Did you install all the dependencies specified in the README?

As you can see the error is slightly different, you're missing `mtools` (I'm 100% sure i added it).

Also, you don't need to run the build script again. Look into cross/bin and echfs/bin. If they both have the right executables, you can just resume from Limine instead of rebuilding GCC two times all over again.

To resume from Limine just run

```
./scripts/build-limine.sh
rm -rf cross/build
```

, as these are the last two lines in the build script.

**Comment by CaydendW** [comment]
I assume this is fixed in pull #37?

**Comment by Chai112** [comment]
I see tysm, ill try it tmr

**Comment by unsafecast** [comment]
> I assume this is fixed in pull #37?

The issue is specific to pr #37

**Comment by CaydendW** [comment]
Rewrite happened. Issue closed
